at_lookup/lib/src/at_lookup_impl.dart

```diff
@@ -36,7 +36,7 @@ class AtLookupImpl implements AtLookUp {
 
   var privateKey;
 
-  var _cramSecret;
+  var cramSecret;
```

Comment:
why has this var made public ?

Reply:
To make it visible in the AtLookupSync class (Child class of this class).